### DISCUSSION TITLE
How to access SOC dashboard

### DISCUSSION THREAD

**Discussion by Lex1020** [Security-Onion-Solutions/securityonion#3904] [Unsupported Versions] [answered]
I'm a bit new to Security Onion so please bear with me. 

I just installed the latest ISO of security onion.  I ran the setup twice and put the server in production mode. I ran sudo so-allow and marked option a and set my IP range.

I can connect to my security onion through HTTPS from a remote machine but when I do all I get is the Readme page.  

I tried running so-analyst-install  but got "command not found"

Am I missing a step? According to the security onion docs after running the so-allow I should be able to reach the Security Onion Dashboard, but all I get is the readme page. 

https://docs.securityonion.net/en/2.3/post-installation.html#post-installation

EDIT:  I seem to be using an old ISO. (16.04 ISO).  I'm downloading the new ISO (2.3.4) now and will try again.  This was probably the issue.

**Comment by weslambert** [comment, marked as answer]
From OP:

_EDIT: I seem to be using an old ISO. (16.04 ISO). I'm downloading the new ISO (2.3.4) now and will try again. This was probably the issue._
